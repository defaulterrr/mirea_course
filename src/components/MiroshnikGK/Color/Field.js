import React from 'react'

const field = (props) => {
    return (
        <div className='inputGK' >
            <input type="text" value={props.value} onChange={props.changed} />
        </div >
    );
}
export default field;