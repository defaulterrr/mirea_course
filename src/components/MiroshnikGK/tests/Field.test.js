import React from 'react';
import { create } from 'react-test-renderer';
import Field from '../Color/Field';

describe('[RENDER] Field â€“ Snapshot test', () => {
  test('Matches the snapshot', () => {
    const field = create(<Field />);
    expect(field.toJSON()).toMatchSnapshot();
  });
});
