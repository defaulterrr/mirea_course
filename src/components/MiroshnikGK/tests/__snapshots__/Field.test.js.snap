// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[RENDER] Field â€“ Snapshot test Matches the snapshot 1`] = `
<div
  className="inputGK"
>
  <input
    type="text"
    value=""
  />
</div>
`;
